modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -10,7 +9,7 @@
 
 A _taint_ allows a node to refuse pod to be scheduled unless that pod has a matching _toleration_.
 
-You apply taints to a node through the node specification (`NodeSpec`) and apply tolerations to a pod through the `Pod` specification (`PodSpec`). A taint on a node instructs the node to repel all pods that do not tolerate the taint.
+You apply taints to a node through the `Node` specification (`NodeSpec`) and apply tolerations to a pod through the `Pod` specification (`PodSpec`). A taint on a node instructs the node to repel all pods that do not tolerate the taint.
 
 Taints and tolerations consist of a key, value, and effect. An operator allows you to leave one of these parameters empty.
```

Comment:
We've opted to capitalize Operator by itself: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#operator

Reply:
That is not a reference, I think, to the 4.x Operators concept. An operator is part of the toleration coding: `operator: "Exists"`. I will look into what I meant here.